app/src/main/res/layout/recycler_row_message_received.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--Layout for any message received by the current user but not sent by them-->
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:id="@+id/layout_message"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+
+    android:background="@drawable/text_background_rounded_corners"
+    android:backgroundTint="@color/message_color_receiver"
+
+    android:padding="8dp"
+
+    android:layout_marginEnd="80dp"
+
```

Comment:
most other xml files don't have extra lines, you can get rid of them to match the other formatting

Reply:
ngl I want to focus on the database rn but I can make an issue if we actually care lol 
update: not bothering